lib/papelillo/mailer_list.ex

```diff
@@ -175,4 +206,27 @@ defmodule Papelillo.MailerList do
     #{deps}
     """
   end
+
+  def validate_email(email) do
+    if EmailChecker.Check.Format.valid?(email) do
+      email
+    else
+      raise "#{email} has not valid email format"
+    end
+  end
+
+  def validate_email(email, config) do
```

Comment:
@benitezhm @bulld0zer this is the new part, you don't need to not-config "domain" in config files, only you need to send a valid email if you want to use another domain besides of default